patches/minecraft/net/minecraft/util/Util.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/util/Util.java
++++ b/net/minecraft/util/Util.java
+@@ -200,6 +200,7 @@
+       try {
+          type = DataFixesManager.func_210901_a().getSchema(DataFixUtils.makeKey(SharedConstants.func_215069_a().getWorldVersion())).getChoiceType(p_240990_0_, p_240990_1_);
+       } catch (IllegalArgumentException illegalargumentexception) {
++         if (!net.minecraftforge.fml.loading.FMLConfig.disableDatafixerWarning())
+          field_195650_a.error("No data fixer registered for {}", (Object)p_240990_1_);
```

Comment:
better fix, as logging already has a config file {the log4j config} change the 'error' to 'debug' and call it good. None of the other things are needed.

Reply:
Sure, I can do that, though I don't see how this is useful. Most of the time people will be looking at the debug log which would still contain those lines and the log4j config is not exactly "user servicable", especially considering the fact that certain "launchers" still fuck around with the logging config.

Reply:
Yes, but these things DO need to be logged. Modders should be addressing them.
We're not caring about users in the debug log. Users don't give a crap if thats spammy or not. Just look at all the popular mods that cause dozens of stacks to be dumped every run.
Honestly, I dont think this message is that big of a deal. Especially not worth having a custom config for.

Reply:
Fair enough